source/extensions/tracers/zipkin/zipkin.pb.cc

```diff
@@ -0,0 +1,2705 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
How come you are checking in generated code? Ideally we use our `*_proto_library` rules to generate this on-the-fly, we don't have any pregenerated stubs elsewhere in Envoy.